### DISCUSSION TITLE
研究：試用GitHub自建的討論區

### DISCUSSION THREAD

**Discussion by lotem** [rime/home#674] [General]
評估是否可以取代Gitter做Rime網站配套的討論區。

按主題組織起來的討論，或許比聊天室的消息流更易讀、方便檢索信息？

**Comment by markwu** [comment]
Discussion 和 Issue 有什麼不同？

**Reply by markwu** [reply]
試試看自己回覆給自己！

**Reply by lotem** [reply]
我認爲區別在於Issues用來追蹤軟件的故障或者具體的開發任務，他最終有一個結案的狀態，表示已經解決問題或完成任務，工單不需要繼續關注了。
Disucussion則適合不設定具體任務、可以持續討論的話題，如規劃、設想、針對用法的問答等等。

**Reply by markwu** [reply]
而且階層比較清楚，看起來很不錯。

**Comment by lotem** [comment]
現在有很多工單在諮詢軟件的用法、配置方法。解答問題後就會關閉工單，不利於知識的積累。
在討論區做問答更合適。

**Comment by ping-wu** [comment]
目前 Gitter 太乱了，根本不知道怎么追索。  GitHub 的 Issues 比较方便。

**Reply by nb5p** [reply]
赞同，Gitter不方便检索

**Comment by lotem** [comment]
使了 giscus 应用替换 gitter 做项目网站的留言系统。
